app/models/article.rb

```diff
@@ -30,6 +30,10 @@ class Article < ApplicationRecord
   counter_culture :user
   counter_culture :organization
 
+  MAX_USER_MENTIONS = 7 # Explicitly set to 7 to accommodate DEV Top 7 Posts
+  # The date that we began limiting the number of user mentions in an article.
+  MAX_USER_MENTION_LIVE_AT = Time.utc(2021, 4, 7).freeze
```

Comment:
I chose this somewhat randomly -- I was operating under the assumption that if this were merged tomorrow, we could set this to be live on Wednesday at midnight UTC (which would be tomorrow/Tuesday, April 6th at 4pm PT / 7pm ET)